frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -55,6 +55,21 @@ if Device:setDateTime() then
     common_settings.time = {
         text = _("Time and date"),
         sub_item_table = {
+          {
+            text = _("12-hour clock"),
+            keep_menu_open = true,
+            checked_func = function()
+                return G_reader_settings:isTrue("twelve_hour_clock")
+            end,
+            callback = function()
+                local twelve_hr = G_reader_settings:isTrue("twelve_hour_clock")
+                if twelve_hr then
```

Comment:
The variable is only used once and it doesn't strike me as a case of increasing clarity.
```suggestion
                if G_reader_settings:isTrue("twelve_hour_clock") then
```